CHANGELOG.md

```diff
@@ -3,6 +3,16 @@
 This is only used for recording changes for major version bumps.
 More minor changes may optionally be recorded here too.
 
+## 52.0.0
+
+* Deprecate the following unused `redis_client` functions:
+  - `redis_client.incr`
```

Comment:
Looks like `incr` is being used [in api](https://github.com/alphagov/notifications-api/blob/81a79e56ced891ad58901ea61ce707e907de3363/app/notifications/process_notifications.py#L160)

Reply:
ah good spot - will change some search terms and look again